Vector/Utils/VectorDesignValues.h

```diff
@@ -31,3 +38,13 @@
 
 // to update the navigation bar buttons color
 // [[AppDelegate theDelegate] recentsNavigationController].navigationBar.tintColor = [UIColor greenColor];
+
+/**
+ Return a background color for an image resource.
+ 
+ @param imageName the image resource name
+ @return the UIColor
+ */
++ (UIColor*)getBackgroundColor:(NSString*)imageName;
```

Comment:
I suggest moving this method in MXKTools and renaming this method with: "convertImageToPatternColor"

" Return the pattern color which can be used to define the background color of a view in order to display the provided image as its background.

Note: If possible add view size as parameter.